hackathon_site/hardware/serializers.py

```diff
@@ -89,6 +88,111 @@ def get_team_code(obj: Order):
         return obj.team.team_code
 
 
+class IncidentCreateSerializer(serializers.Serializer):
```

Comment:
This is waaaaaay more than you need. I realize it was copy-pasted from the Order serializer, but why? Orders was an extremely custom use case (that involved creating a bunch of `OrderItems` under the hood). Creating an Incident is very straightforward, and just requires a barebones serializer that looks like:

```python
class IncidentCreateSerializer(serializers.ModelSerializer):
    class Meta:
        model = Incident
        fields = (
            "id",
            "state",
            "time_occurred",
            "description",
            "order_item",
            "created_at",
            "updated_at",
        )
```

I forget how nested serializers are handled during create calls, you might actually be able to just tuse the pre-existing `IncidentSerializer` and just add `read_only_fields = ("team_id",)` to the Meta class.